packages/protocol/thread/thread.ts

```diff
@@ -883,6 +899,23 @@ class _ThreadFront {
     );
   }
 
+  findMessagesInRange(range: PointRange) {
+    return client.Console.findMessagesInRange({ range }, ThreadFront.sessionId!);
+  }
+
+  async findKeyboardEvents(onKeyboardEvents: (events: keyboardEvents) => void) {
+    const keyboardEventsLoaded = client.Session.findKeyboardEvents({}, this.sessionId!);
+    client.Session.addKeyboardEventsListener(onKeyboardEvents);
+    return keyboardEventsLoaded
```

Comment:
The order of these two should probably be reversed (add the handler before calling `findKeyboardEvents`)
```suggestion
    client.Session.addKeyboardEventsListener(onKeyboardEvents);
    return client.Session.findKeyboardEvents({}, this.sessionId!);
```
Might not matter in this case but I think it's a good general practice to follow (so you don't miss any potentially synchronous events).

Reply:
PS Surprised the linter didn't complain about the missing semicolon on line 909